CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `AstarteDeviceSdk`.
 - Trait `FromEvent` to convert a generic object aggregate into a Rust struct.
 - Implementation of the connection over GRPC to the message hub.
+- Add many interfaces at once with `extend_interfaces`
+  [#293](https://github.com/astarte-platform/astarte-device-sdk-rust/pull/293)
```

Comment:
Move this change to  `## [Unreleased]` tag

Reply:
Done